src/Model/Event.php

```diff
@@ -363,11 +362,10 @@ public static function getItemArrayForId(int $event_id, array $item = []): array
 		$item['allow_gid']     = $event['allow_gid'];
 		$item['deny_cid']      = $event['deny_cid'];
 		$item['deny_gid']      = $event['deny_gid'];
-		$item['private']       = intval($event['private'] ?? 0);
+		$item['private']       = $event['allow_cid'] && $event['allow_gid'] && $event['deny_cid'] && $event['deny_gid'] ? 0 : 1;
 		$item['visible']       = 1;
 		$item['verb']          = Activity::POST;
 		$item['object-type']   = Activity\ObjectType::EVENT;
-		$item['post-type']     = Item::PT_EVENT;
```

Comment:
Why is this removed?

Reply:
Array key is duplicated in the same block.